docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -127,17 +150,72 @@ PAF-of-1 Cause/Risk Pairs
 Risk-Outcome Relationships
 ++++++++++++++++++++++++++
 
-Coverage Gap Framework
-++++++++++++++++++++++
+Stratifying Exposure Variables by Baseline Intervention Coverage
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+From GBD we obtain overall population estimates for our outcomes of interest
+(prevalence of vitamin A deficiency, birth prevalence of neural tube defects,
+distribution of haemoglobin levels, and distribution of birth weights). Because
+there exists baseline coverage of our interventions --- i.e. there is already
+some food fortification occurring in the locations we are simulating --- these
+overall population estimates include both people who are eating fortified food
+and those who are not.
+
+When we introduce additional food fortification into the population, only those
+who are *not* already eating fortified food should receive the benefit.
+Therefore, in order to properly measure the effects of our interventions, we
+must first stratify our population into those who are already covered vs. not
+covered by the forticant of interest. We then need to calculate the risk
+exposure (or other relevant outcome) in each coverage stratum to get a complete
+picture of the baseline population. For example, we will need to know the
+prevalence of vitamin A deficiency or the distribution of haemoglobin in the
+subpopulation already eating fortified food as well as in the subpopulation not
+eating fortified food.
+
+These coverage-stratified exposures are calculated so that:
+
+1.  Their population-weighted average matches the overall population estimate
+    from GBD, and
+2.  The differential exposure between the subpopulations matches the
+    corresponding effect size for our intervention.
+
+For example, the ratio of the prevalence of vitamin A deficiency between the
+fortified and unfortified groups should match the risk ratio of vitamin A
+fortification on vitamin A deficiency from the literature, and the difference in
+mean haemoglobin level between the fortified and unfortified groups should match
+the mean shift in haemoglobin from iron fortification found in the literature.
+When we stratify the population in this way, we are assuming that the effect
+sizes of fortification found in the literature are generalizable to the
+populations we are simulating (note that if this assumption is *not* true, we
+should not be simulating the interventions in theses populations at all).
```

Comment:
We are assuming that the proportion of our outcome attributable to our exposure is the same in our model population as it is in our study population. For something like vitamin A deficiency, it's pretty much all attributable to vitamin A intake. For something like anemia, it is *not* necessarily all attributable to iron intake (i.e. non-iron responsive anemias). The assumption is valid for situations in which there are not many other factors that cause the outcome other than the exposure at hand. However, it should be more carefully evaluated when there are multiple factors that can cause the outcome (*many* things can cause low birth weight).

We generally get around this issue in our vivarium simulations by directly using the PAF for a given exposure and outcome. However, we don't use the PAF in this approach, so that is why it requires stronger assumptions.

I don't know that we need to write this here, but I don't think we've written it down, so I just wanted to make a note : )

Reply:
Thanks for this explanation, @alibow !  When you say we don't use the PAF in this approach, are you referring specifically to continuous outcomes like hemoglobin or birth weight?  Because I thought our approach for dichotomous outcomes like VAD was essentially equivalent to computing the PAF.  Is that accurate, or am I missing something?

Reply:
Ah... yes, the point I was trying to make was that we're not using GBD's PAF. GBD's PAFs are calculated after going through a process that vets the RRs and adjusts for potential confounders etc. in MRBRT and assesses generalizability so we are relying on this analysis and don't have to make the assumption on our own; we just have to assume that GBD is correct : ).

Reply:
I thought we are using GBD's paf and that's why we run into limitations cuz we are using an RR from one study population and the PAF from our country location from GBD?

Reply:
I don't think we have GBD PAF's for the RR's we get from external literature for our interventions. Is there a specific example you're thinking of @Ninicorn ?

Reply:
@alibow I incorporated your description above, and added some additional comments in a todo.  I'm still a bit hazy about what exactly our assumptions mean and when they are appropriate, so I think it's worth continuing to flesh this out as much as possible.